content/02.introduction.md

```diff
@@ -1,41 +1,52 @@
+Morever, domain-specific perspectives and resources providing tractable examples and practical guidance for researchers in ecology and evolution on GitHub are scarce (but see CITE, CITE, CITE).
+An increased availability of data and code managenent standards -- of which GitHub is one increasingly important component -- make research more reproducible and collaborative [@doi:10.1002/bes2.1801].
+More importantly, a widespread, common adoption of GitHub for collaborating on a variety of research tasks can ultimately enable EEB researchers to spend less time on creating novel processes for collaboration and more time on their scientific research [@doi:10.3897/rio.6.e56508].
 
-### What's missing about GitHub in EcoEvo and our objective
+### Our objective
 
-*Contributors to this section: RCO*
+*Contributors to this section: RCO, PHPB*
 
-Our manuscript is the result of a hackathon held during the 2021 conference for the [Society for Open, Reliable, and Transparent Ecology and Evolutionary Biology (SORTEE)](https://www.sortee.org/).
-We convened a group of ~30 EEB researchers with varying levels of familiarity with using GitHub as part of their research.
-During the hackathon, we identified a need for a more friendly introduction to that ways EEB researchers can start using GitHub to make their research more collaborative and transparent.
-While there have been numerous calls for more researchers in the natural sciences to leverage GitHub's features [@doi:10.1038/538127a], few resources provide tractable examples and practical guidance for first-time GitHub users.
-Simple data and code manage habits (of which GitHub is one increasingly important component) make research more reproducible and collaborative (Alston and Rick 2021).
-Most importantly, leveraging GitHub for collaborating on a variety of research tasks, with others who do the same, can ultimately enable EEB researchers to spend less time on creating novel processes for collaboration and more time on their scientific research [@doi:10.3897/rio.6.e56508].
+This manuscript is the result from an academic hackathon held during the 2021 conference for the [Society for Open, Reliable, and Transparent Ecology and Evolutionary Biology (SORTEE)](https://www.sortee.org/).
+We convened a group of ~30 EEB researchers with varying levels of familiarity with using GitHub as part of their research projects to showcase and discuss how existing features can contribute to the documentation and collaboration in ecological and evolutionary research.
+During the hackathon, we identified the need for a formal discussion on how EEB researchers can benefit from GitHub and its features to make their research more collaborative and transparent.
+We outline thirteen practical workflows on GitHub features that can help towards a more collaborative, transparent and reproducible EEB research.
```

Comment:
```suggestion
Here, we outline thirteen practical uses of (?) GitHub features that can help towards a more collaborative, transparent and reproducible EEB research.
```

Reply:
Incorporated! Thank you for the recommendation!